docs/network-upgrades.md

```diff
@@ -0,0 +1,28 @@
+---
+id: network-upgrades
+title: Network upgrades
+sidebar_label: Network upgrades
+---
+
+:::info PUBLIC PREVIEW DOCUMENT
```

Comment:
```suggestion
import DraftExpectationsPartial from '@site/docs/partials/_draft-expectations-partial.md'; 

<DraftExpectationsPartial />
```

Partial views help minimize maintenance overhead when repeated copy needs to be updated.